### DISCUSSION TITLE
402 Payment Required license Key不合法咋回事呢

### DISCUSSION THREAD

**Discussion by yunsong1** [JimmyLv/BibiGPT#115] [Q&A]
![Snipaste_2023-05-06_20-45-22](https://user-images.githubusercontent.com/113420956/236624938-d7d38efc-15e8-481f-8816-6a7ba6abaacc.png)
为啥我填了API key一直显示不合法，需要付款

**Comment by JimmyLv** [comment]
现在应该都可以啦！可以试试哦
